examples/create_pod/main.c

```diff
@@ -0,0 +1,118 @@
+#include <apiClient.h>
+#include <CoreV1API.h>
+#include <malloc.h>
+#include <stdio.h>
+#include <errno.h>
+
+#define K8S_APISERVER_BASEPATH "https://9.111.254.254:6443"
```

Comment:
Let's anonymize this with something like `"https://your.server.here"`

Reply:
OK.

Reply:
This doesn't seem to have been fixed?

Reply:
Yes. I have updated but not pushed to Github.

I want to 
1. Prepare a GNU Indent parameter list first and get your reviewed
2. Use GNU Indent to check my new source 
4. Push the new source code to here for review again.